prefs.js

```diff
@@ -31,6 +31,26 @@ const TimezoneExtensionPrefsWidget = new GObject.Class({
                                  use_markup: true, margin_bottom: 6,
                                  hexpand: true, halign: Gtk.Align.START}));
 
+        let box = new Gtk.Box({orientation: Gtk.Orientation.HORIZONTAL, spacing: 5});
+
+        box.add(new Gtk.Label({ label: '<b>Highlight working hours</b>',
```

Comment:
You missed the checkbox that will enable/disable this option.
If the checkbox/property is disabled (default value), things must not change, i.e., we keep current behavior of highlighting nothing at all.
If it is enabled, then you apply the logic to highlight (green/red) the time label.

Also, that sentence could be: Highlight working hours: From ___ to ___